generators/generator-microsoft-bot-adaptive/baseGenerator.js

```diff
@@ -48,4 +50,28 @@ module.exports = class extends Generator {
       `${integration} is not a registered integration`
     );
   }
+
+  copyBotTemplateFiles({ path = ['**', '*.*'], templateContext = {} } = {}) {
```

Comment:
I believe that yeoman uses prototype reflection to determine tasks with the convention that non-task methods must begin with `_` - not sure if this is causing problem or not, but it might be worth prefixing the name with an underscore.

Reply:
No issues encountered so far - I *think* it might only be within the specified class type itself? That said, I'll do it for good measure since I'm not 100% sure.